MiniProject1.py

```diff
@@ -0,0 +1,96 @@
+# Kimber's file
+from pattern.en import *
+rhymeWords=dict() #dictionary of words that rhyme
+generalWords=dict() #markov chain dictionary
+#fo = open("ShakespeareSonnets.txt", "r")
+with open('ShakespeareSonnets.txt', 'r') as f:
+	read_data = f.read()
+f.closed
+sonnetz= read_data.split('\n\n')
+sonnetLine=[0, 1, 2, 3,4, 5,6 ,7, 8, 9, 10, 11, 12, 13]
```

Comment:
A comment explaining what this line is for would be useful. In all fairness, you do a pretty good job of explaining your code with comments for most of the rest of everything :^)